tests/smc/test_tempered_smc.py

```diff
@@ -68,8 +68,8 @@ def logprior_fn(x):
         iterates = []
         results = []  # type: List[TemperedSMCState]
 
-        hmc_kernel = blackjax.hmc.kernel()
-        hmc_init = blackjax.hmc.init
+        hmc_kernel = blackjax.mcmc.hmc.kernel()
```

Comment:
IIUC, in the top level __init__.py you imported the symbol, so you dont need to make this change